doc: Add note about using all zeros JoinEUI (#590)
TheThingsIndustries/lorawan-stack-docs: doc/content/devices/adding-devices/_index.md

Enter a **JoinEUI/AppEUI** if provided by your manufacturer. If your device is programmable, you may use the **Fill with zeros** button, and then program the same JoinEUI/AppEUI (`0000000000000000`) in the device.

{{< note >}} Some devices do not support using `0000000000000000` as a JoinEUI/AppEUI, because technically, this value is invalid. However, {{% tts %}} supports using this value to indicate the absence of an actual JoinEUI/AppEUI.

> Assuming your device does not support using `0000000000000000` as a JoinEUI/AppEUI, if you try using it, you may encounter an error resembling to `LoRa parameters are invalid`. Try using the DevEUI value as a JoinEUI/AppEUI. {{</ note >}}

Reviewer: ```suggestion
If your device gives an error when using `0000000000000000` as a JoinEUI/AppEUI, try using the DevEUI value as a JoinEUI/AppEUI, both in {{% tts %}} and on your device. {{</ note >}}
```
